scripts/ci-test.sh

```diff
@@ -69,24 +72,23 @@ oracletest() {
   npm i strong-oracle
   # Check for node_modules/strong-oracle explicitly because even if installation of it fails, npm doesn't catch it.
   if [[ ! -d node_modules/strong-oracle ]] ; then echo 'node_modules/strong-oracle is not there, return error.' ; exit 1 ; fi
-  # Build container
-  LEDGER_UNIT_DB_URI='oracle://system:oracle@localhost:49161/' LD_LIBRARY_PATH=/opt/oracle/instantclient node node_modules/.bin/istanbul test -- _mocha
+  npm run setup-oracle
+  npm run test-oracle
 }
 
 oneNode() {
   lint
   dockerBuild
   sqlitetestest
   integrationtest
-  mysqltest
```

Comment:
I don't think we have much choice since mysql does not enforce CHECK constraints, and the approach we used to fix this critical bug (money creation/destruction) depends on a CHECK constraint.

I'll post to the five-bells slack channel to see if anyone has any other ideas.